src/main/java/com/coveo/saml/ValidatorUtils.java

```diff
@@ -34,6 +42,7 @@ private static void validateResponse(StatusResponseType response, String respons
     try {
       new ResponseSchemaValidator().validate(response);
     } catch (SamlException e) {
+      logger.debug("The response schema validation failed", e);
```

Comment:
I know this is debug but this usually means this will be double logged since the throwable is already wrapped in another. Any specific reason why you added this?

Reply:
Good point. I think we can remove that.